eZ/Publish/Core/REST/Server/Input/Parser/Criterion/Field.php

```diff
@@ -24,10 +39,50 @@ class Field extends BaseParser
      *
      * @throws \eZ\Publish\Core\REST\Common\Exceptions\Parser
      *
-     * @return \eZ\Publish\API\Repository\Values\Content\Query\Criterion\Field
+     * @return FieldCriterion
      */
     public function parse(array $data, ParsingDispatcher $parsingDispatcher)
     {
-        throw new \Exception('@todo implement');
+        if (!\array_key_exists('Field', $data)) {
```

Comment:
We don't place namespace separators on PHP functions, and for classes our PHP CS strongly prefer importing the name.

Reply:
Thanks!
Fixed both things. From what I know, it is different in V2 (platform-ui-bundle?), hence mistakes :D